mathicsscript/inputrc-no-unicode

```diff
@@ -29,728 +29,3 @@ $include "inputrc"
 "\e\\chi\e": "\\[Chi]"
 "\e\\psi\e": "\\Psi]"
 "\e\\omega\e": "\\[Omega]"
-
-# Autogenerated from https://github.com/Mathics3/mathics-development-guide/blob/master/resources/named-characters-data.csv
-"a'": "\\[AAcute]"
```

Comment:
@GarkGarcia  All of these characters should be regenerated.  They are missing the escape at the beginning and end of the escape sequence. 

For example `"\ea'\e"` rather than `"a'"`.

Reply:
Ohh, yes, forgot about that. @rocky btw we could use `mathics-scanner` to proceduraly generate the inputrc files now. It would look something like the following:

```py
from mathics_scanner.characters import replace_wl_with_plain_text, aliased_characters

# Print inputrc with unicode
for alias in alieased_characters:
  print(f'"\\e{escape(alias)}\\e": "{escape(replace_wl_with_plain_text(aliased_characters[alias]))}"')

# Print inputrc without unicode
for alias in alieased_characters:
  print(f'"\\e{escape(alias)}\\e": "{escape(replace_wl_with_plain_text(aliased_characters[alias], use_unicode=False))}"')
```

Reply:
Great! Make it so!

Reply:
> Great! Make it so!

Ok, I just need to know which characters should be escaped inside of a inputrc string.